src/app/Main/Body/QueryPage/QueryPageContent/Parameter/ParameterList-Layout.component.tsx

```diff
@@ -9,10 +9,24 @@ const listStyle = css`
 const listHeaderStyle = css`
     position: relative;
 `;
+const spaceStyle = css`
+    padding: 0;
+    margin: 0;
+`;
+const parameterValue = css`
+    margin-top: 0;
+    margin-left: 2em;
+    span {
+        letter-spacing: 0;
+    }
+`;
 
 export interface ListProps {
     parameterLabel: string;
-    parameterValuesList: JSX.Element[];
+    parameterNames: {
+        parameterName: string | JSX.Element;
```

Comment:
As discussed sometimes ago: In general it would be better to pass data here only and let this component create the actual JSX.Elements for the names. So the involved services would not depend on react and just generate the necessary data. Then this component would be a clean view again (now it is some mix of a view and a layout).
But this would be something for a separat ticket.